src/lib.rs

```diff
@@ -1224,6 +1226,8 @@ impl TestKitApi {
             resp.expect("Got unexpected `Err(..)` response")
         };
 
+        debug!("GET request: {} Response: {:?}", url, resp);
```

Comment:
Some time ago we agreed not to use `debug` log level. Argumentation was the following: when one wants to debug some functionality one can add such logging and easily filter it in the whole project.